code/game/atoms.dm

```diff
@@ -390,24 +390,54 @@
 
 /atom/proc/build_reagent_description(mob/user)
 	. = list()
-	if(reagents)
+	if(!reagents)
+		return
+	var/one_percent = reagents.total_volume / 100
+	var/blood_type = ""
+	if(user.advanced_reagent_vision())	// You can see absolute unit concentrations in transparent containers and % concentrations in opaque containers. You can also see blood types.
+		. += "<span class='notice'>It contains:</span>"
+		if(!reagents.reagent_list.len)
```

Comment:
![image](https://github.com/ParadiseSS13/Paradise/assets/77684085/9f32db00-4fb4-4814-9577-97d08d30b8d3)

Any reason why you changed it making it use .len?

Reply:
That got missed during a merge deconflict, I've changed it back now.